codebundles/aws-c7n-ec2-health/.runwhen/templates/aws-c7n-ec2-health-slx.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: runwhen.com/v1
+kind: ServiceLevelX
+metadata:
+  name: {{slx_name}}
+  labels:
+    {% include "common-labels.yaml" %}
+  annotations:
+    {% include "common-annotations.yaml" %}
+spec:
+  imageURL: https://PLACEHOLDER
+  alias: AWS EC2 Health For Region {{match_resource.resource.region}} in Account {{match_resource.resource.account_id}}
+  asMeasuredBy: The number of unused and unpatched EC2 instances in region {{match_resource.resource.region}} and account {{match_resource.resource.account_id}}
```

Comment:
I don't think "unused" is valid here. How are you to assume it's unused? I think the word we want to go with is "stale".